charts/pravega-operator/README.md

```diff
@@ -44,6 +45,8 @@ $ helm uninstall [RELEASE_NAME]
 
 This command removes all the Kubernetes components associated with the chart and deletes the release.
 
+>Note: If you want to delete the Pravega clusters, make sure to do it before uninstalling the operator.
```

Comment:
It would be good to explain the consequence of having an orphan "Pravega cluster".
I think we recently had a discussion around enforcing a check on not allowing uninstall on p-operator, if at-least 1 cluster it manages is still around. Please check with @SrishT on this..

Reply:
Pre-delete hook handles this. Operator will get uninstalled only if the cluster is deleted.  Removed the Note